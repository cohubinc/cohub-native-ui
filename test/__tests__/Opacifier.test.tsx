import React from "react";
import { mount } from "enzyme";

import { Opacifier } from "dist";

describe("Opacifier", () => {
  it("renders without crashing", async () => {
    const wrapper = mount(<Opacifier />);

    expect(wrapper).toMatchSnapshot();
  });
});
