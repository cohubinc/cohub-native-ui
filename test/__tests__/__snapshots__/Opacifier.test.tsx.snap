// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Opacifier renders without crashing 1`] = `
<Opacifier>
  <div
    style={
      Object {
        "height": "initial",
        "overflowY": "scroll",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "background": "linear-gradient(0deg, rgba(250, 250, 250, 1) 0%, rgba(250, 250, 250, 0) 100%)",
          "bottom": 0,
          "height": "25%",
          "position": "sticky",
          "width": "100%",
        }
      }
    />
  </div>
</Opacifier>
`;
